arm-asm-s2s/arm-asm-s2s.json

```diff
@@ -0,0 +1,171 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
Please add the "description" tags for the parameters. See other templates in the repo